pollination/annual_daylight/entry.py

```diff
@@ -129,6 +129,30 @@ def create_octree(self, model=create_rad_folder._outputs.model_folder):
             }
         ]
 
+    @task(
+        template=SplitGridFolder, needs=[create_rad_folder], 
+        sub_paths={'input_folder': 'grid'}
+    )
+    def split_grid_folder(
+        self, input_folder=create_rad_folder._outputs.model_folder,
+        grid_count=cpu_count, sensor_count=1000
```

Comment:
Maybe we should leave this as `sensor_count=200` in order to remain consistent with the old behavior.